static/src/javascripts/projects/commercial/modules/dfp/dfp-api.spec.js

```diff
@@ -473,6 +475,7 @@ describe('DFP', () => {
             onConsentChange.mockImplementation(callback =>
                 callback(tcfv2WithConsent)
             );
+            getConsentFor.mockReturnValue(true);
```

Comment:
I don’t think `getConsentFor` should be mocked

If the `onConsentChange` mock works, the state it passes to the callback should have the correct consents.

Reply:
I tried that but there were some issues using a mixed version of mock and actual implementation from consent-management-platform using `requireActual` from jest. 
In particular we will need to add support of ES2020 inside our testing framework as there are errors with transpiling when using the actual implementation. Maybe worth to create a card for fixing that in the future. cc @sndrs 

On another note, I think it's OK (at least for now) to mock it since the actual functionality of the `getConsentFor` is tested inside the `cmp` library itself. You could argue though what's the point of mocking the `onConsentChange` with correct test data if it doesn't matter what we are passing in the `getConsentFor.` Maybe I should clean up the test data but though to leave them in for clarity.